extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -397,6 +412,12 @@ export class SchemaCompareDialog {
 			};
 		});
 
+		if (idx >= 0) {
```

Comment:
Suggest putting a comment here explaining that you're moving it to the top of the list so it's the default selected item